feat(HTTP): Refresh 4XX response pages (#34620)
mdn/content: files/en-us/web/http/status/411/index.md

{{HTTPSidebar}}

The HTTP **`411 Length Required`** status response code is part of the `400`-`499` class of [client error responses](/en-US/docs/Web/HTTP/Status#client_error_responses) and indicates that the server refuses to accept the request without a defined {{HTTPHeader("Content-Length")}} header.

> **Note:**

> > When sending data in a series of chunks, the `Content-Length` header is omitted and at the beginning of each chunk you need to add the length of the current chunk in hexadecimal format.

Reviewer: ```suggestion
> When sending data in a series of chunks, the `Content-Length` header is omitted, and at the beginning of each chunk, the length of the current chunk needs to be included in hexadecimal format.
```

Author: done at https://github.com/mdn/content/pull/34620/commits/5ec35dbdd45b72e904277ceab6f6106a745a6c10#diff-347b1d715b6df7331d934826e850695e11ec5877f37c8e56e4b5963ca92fdd76R14

> See {{HTTPHeader("Transfer-Encoding")}} for more details.

## Status

```http
411 Length Required
```

## Examples

### Chunked POST request

> The following request is sent chunked, which can be the default method of sending data in some cases (specifically when [writing to streams](https://nodejs.org/api/http.html#requestwritechunk-encoding-callback)):

Reviewer: ```suggestion
The following request is sent chunked, which is the default method of sending data in some cases, such as when [writing to streams](https://nodejs.org/api/http.html#requestwritechunk-encoding-callback):
```

Author: done at https://github.com/mdn/content/pull/34620/commits/5ec35dbdd45b72e904277ceab6f6106a745a6c10#diff-347b1d715b6df7331d934826e850695e11ec5877f37c8e56e4b5963ca92fdd76R27
